### ISSUE TITLE
Super IP blacklist

### ISSUE THREAD

**Issue by Windows-i7** [TotalFreedom/TotalFreedomMod#422]
Some IPs should not be supered, as they are public. In particular, this one has been a recurring problem: 167.114.97.16 (MC connect). I apologize if this duplicates an issue, but I did not see it in here when I checked.

**Comment by zD12** [comment]
I don't think this is such a good idea.  This will only lag the server running TotalFreedomMod once hundreds, if not, thousands of proxies, VPN's, and otherwise public IP addresses are added.  There isn't any need to code this in, instead, administrators should use common sense and follow the policies outlined on the forums.

**Comment by JeromSar** [comment]
@zD12 The feature on its own is a good idea. It's how we use it that might make a difference. Then again, you shouldn't be worrying about lag, since this check is only performed when adding a super admin is negligible.
